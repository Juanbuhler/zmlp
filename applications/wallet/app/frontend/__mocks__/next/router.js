/**
 * on
 */

let mockOnFunction = () => {}

export const __setMockOnFunction = fn => {
  mockOnFunction = fn
}

/**
 * off
 */

let mockOffFunction = () => {}

export const __setMockOffFunction = fn => {
  mockOffFunction = fn
}

/**
 * Router
 */

const Router = {
  events: {
    on: (...args) => {
      mockOnFunction(...args)
    },
    off: (...args) => {
      mockOffFunction(...args)
    },
  },
}

export default Router
