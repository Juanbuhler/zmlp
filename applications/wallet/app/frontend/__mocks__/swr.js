import { createElement } from 'react'

const noop = () => () => {}

/**
 * <SWRConfig />
 */

export const SWRConfig = ({ children, ...rest }) =>
  createElement('SWRConfig', rest, children)

/**
 * useSWR
 */

let mockUseSWRResponse = {}

export const __setMockUseSWRResponse = data => {
  mockUseSWRResponse = { revalidate: noop, ...data }
}

const useSWR = () => {
  return mockUseSWRResponse
}

export default useSWR
