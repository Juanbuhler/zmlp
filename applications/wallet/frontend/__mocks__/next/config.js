const getConfig = () => ({
  serverRuntimeConfig: process.env,
  publicRuntimeConfig: process.env,
})

export default getConfig
