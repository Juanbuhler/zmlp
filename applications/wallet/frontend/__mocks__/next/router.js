/**
 * useRouter
 */

let mockUseRouter = {
  query: {
    action: '',
  },
}

export const __setUseRouter = data => {
  mockUseRouter = data
}

export const useRouter = () => mockUseRouter

/**
 * push
 */

let mockPushFunction = () => {}

export const __setMockPushFunction = fn => {
  mockPushFunction = fn
}

/**
 * on
 */

let mockOnFunction = () => {}

export const __setMockOnFunction = fn => {
  mockOnFunction = fn
}

/**
 * off
 */

let mockOffFunction = () => {}

export const __setMockOffFunction = fn => {
  mockOffFunction = fn
}

/**
 * Router
 */

const Router = {
  push: (...args) => {
    mockPushFunction(...args)
  },
  events: {
    on: (...args) => {
      mockOnFunction(...args)
    },
    off: (...args) => {
      mockOffFunction(...args)
    },
  },
}

export default Router
