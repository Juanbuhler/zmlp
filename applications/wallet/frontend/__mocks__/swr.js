import { createElement } from 'react'

const noop = () => () => {}

/**
 * <SWRConfig />
 */

export const SWRConfig = ({ children, ...rest }) =>
  createElement('SWRConfig', rest, children)

/**
 * cache
 */

export const { cache } = jest.requireActual('swr')

/**
 * useSWR
 */

let mockUseSWRResponse = { revalidate: noop, mutate: noop }

export const __setMockUseSWRResponse = (data) => {
  mockUseSWRResponse = { revalidate: noop, mutate: noop, ...data }
}

const useSWR = () => {
  return mockUseSWRResponse
}

export default useSWR
