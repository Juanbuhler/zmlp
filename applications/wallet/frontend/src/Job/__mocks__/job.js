const job = {
  id: '223fd17d-7028-1519-94a8-d2f0132bc0c8',
  organizationId: '00000000-9998-8888-7777-666666666666',
  name: 'image_similarity.json',
  type: 'Import',
  state: 'Finished',
  assetCounts: {
    assetCreatedCount: 15,
    assetReplacedCount: 0,
    assetWarningCount: 0,
    assetErrorCount: 6,
  },
  taskCounts: {
    tasksTotal: 3,
    tasksWaiting: 0,
    tasksRunning: 0,
    tasksSuccess: 3,
    tasksFailure: 0,
    tasksSkipped: 0,
    tasksQueued: 0,
  },
  timeStarted: 1579220709790,
  timeUpdated: 1579220816592,
  timeCreated: 1579220705734,
  priority: 100,
  paused: false,
  timePauseExpired: -1,
  maxRunningTasks: 1024,
  jobId: '223fd17d-7028-1519-94a8-d2f0132bc0c8',
  actions: {
    cancel:
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/cancel/',
    errors:
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/errors/',
    'max-running-tasks':
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/max_running_tasks/',
    pause:
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/pause/',
    priority:
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/priority/',
    restart:
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/restart/',
    resume:
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/resume/',
    'retry-all-failures':
      'http://localhost/api/v1/projects/299a5a40-368c-4f7a-8664-36ded98346ef/jobs/223fd17d-7028-1519-94a8-d2f0132bc0c8/retry_all_failures/',
  },
}

export default job
