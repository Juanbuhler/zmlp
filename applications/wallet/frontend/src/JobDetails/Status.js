import PropTypes from 'prop-types'

import ProgressBar from '../ProgressBar'

import { spacing } from '../Styles'

const JobDetailsStatus = ({ taskCounts }) => {
  return (
    <>
      <div
        css={{
          paddingBottom: `${spacing.moderate}px`,
        }}>
        Task Progress:
      </div>

      <ProgressBar taskCounts={taskCounts} containerWidth={444} />
    </>
  )
}

JobDetailsStatus.propTypes = {
  taskCounts: PropTypes.shape({
    tasksFailure: PropTypes.number.isRequired,
    tasksSkipped: PropTypes.number.isRequired,
    tasksSuccess: PropTypes.number.isRequired,
    tasksRunning: PropTypes.number.isRequired,
  }).isRequired,
}

export default JobDetailsStatus
