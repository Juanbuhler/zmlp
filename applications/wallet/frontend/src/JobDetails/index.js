import Head from 'next/head'
import { useRouter } from 'next/router'

import Breadcrumbs from '../Breadcrumbs'

const JobDetails = () => {
  const {
    query: { jobId },
  } = useRouter()

  return (
    <>
      <Head>
        <title>Job Details</title>
      </Head>

      <Breadcrumbs crumbs={['Job Queue', 'Job Details']} />
      <div>Job ID: {jobId}</div>
    </>
  )
}

export default JobDetails
