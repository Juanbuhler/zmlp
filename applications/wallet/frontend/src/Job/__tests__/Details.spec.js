import TestRenderer, { act } from 'react-test-renderer'

import job from '../__mocks__/job'

import JobDetails from '../Details'

const PROJECT_ID = '76917058-b147-4556-987a-0a0f11e46d9b'
const JOB_ID = '82d5308b-67c2-1433-8fef-0a580a000956'

const noop = () => () => {}

describe('<JobDetails />', () => {
  it('should render properly', async () => {
    require('next/router').__setUseRouter({
      query: { projectId: PROJECT_ID, jobId: JOB_ID },
    })

    require('swr').__setMockUseSWRResponse({
      data: { ...job, state: 'InProgress' },
      mutate: noop,
    })

    const component = TestRenderer.create(<JobDetails />)

    expect(component.toJSON()).toMatchSnapshot()

    act(() => {
      component.root
        .findByProps({ 'aria-label': 'Toggle Dropdown Menu' })
        .props.onClick({ preventDefault: noop, stopPropagation: noop })
    })

    await act(async () => {
      component.root.findByProps({ children: 'Pause' }).props.onClick()
    })

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('should render properly with an action confirmation', async () => {
    require('next/router').__setUseRouter({
      query: {
        projectId: PROJECT_ID,
        jobId: JOB_ID,
        action: 'Retry All Failures',
      },
    })

    require('swr').__setMockUseSWRResponse({
      data: job,
      mutate: noop,
    })

    const component = TestRenderer.create(<JobDetails />)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
