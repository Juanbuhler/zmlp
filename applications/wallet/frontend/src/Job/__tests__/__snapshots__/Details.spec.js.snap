// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<JobDetails /> should render properly 1`] = `
.emotion-0 {
  font-size: 17px;
  line-height: 20px;
  font-weight: 500;
  padding-top: 16px;
  padding-bottom: 16px;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: -4px;
  padding-right: 48px;
}

.emotion-3 {
  position: relative;
  height: 100%;
}

.emotion-4 {
  -webkit-flex: 0;
  -ms-flex: 0;
  flex: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  border-radius: 2px;
  padding: 0;
  white-space: nowrap;
  font-size: 15px;
  line-height: 19px;
  font-weight: 500;
  border: none;
  cursor: pointer;
  color: #ffffff;
  background-color: transparent;
  width: 200px;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 16px;
  padding-right: 12px;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-bottom: 4px;
  color: #ffffff;
}

.emotion-4,
.emotion-4:hover,
.emotion-4:visited {
  background-color: #808080;
}

.emotion-4:hover {
  background-color: #b3b3b3;
}

.emotion-4[aria-disabled=true] {
  background-color: #808080;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-right: 48px;
  padding-top: 12px;
  padding-bottom: 12px;
}

.emotion-6 {
  font-weight: 700;
  color: #ffffff;
  padding-bottom: 4px;
}

.emotion-7 {
  color: #b3b3b3;
}

.emotion-20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 16px;
  width: 212px;
  position: relative;
}

.emotion-21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

.emotion-22 {
  height: 100%;
  -webkit-flex: 3 0 auto;
  -ms-flex: 3 0 auto;
  flex: 3 0 auto;
  background-color: #009f22;
}

.emotion-22:first-of-type {
  border-top-left-radius: 4px;
  border-bottom-left-radius: 4px;
}

.emotion-22:last-of-type {
  border-top-right-radius: 4px;
  border-bottom-right-radius: 4px;
}

<div>
  <h3
    className="emotion-0"
  >
    Job: 
    image_similarity.json
  </h3>
  <div
    className="emotion-1"
  >
    <div
      className="emotion-2"
    >
      <div
        className="emotion-3"
      >
        <button
          aria-label="Toggle Dropdown Menu"
          className="emotion-4"
          onBlur={[Function]}
          onClick={[Function]}
          type="button"
        >
          Modify Job
          <svg
            data-file-name="chevron.svg"
            height={20}
          />
        </button>
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        ID
        :
      </div>
      <div
        className="emotion-7"
      >
        82d5308b-67c2-1433-8fef-0a580a000956
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Status
        :
      </div>
      <div
        className="emotion-7"
      >
        InProgress
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Priority
        :
      </div>
      <div
        className="emotion-7"
      >
        100
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        # Assets
        :
      </div>
      <div
        className="emotion-7"
      >
        4
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Progress
        :
      </div>
      <div
        className="emotion-7"
      >
        <div
          aria-label="Progress Bar"
          className="emotion-20"
          onKeyPress={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="button"
          tabIndex="0"
        >
          <div
            className="emotion-21"
          >
            <div
              className="emotion-22"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<JobDetails /> should render properly 2`] = `
.emotion-0 {
  font-size: 17px;
  line-height: 20px;
  font-weight: 500;
  padding-top: 16px;
  padding-bottom: 16px;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: -4px;
  padding-right: 48px;
}

.emotion-3 {
  position: relative;
  height: 100%;
}

.emotion-4 {
  -webkit-flex: 0;
  -ms-flex: 0;
  flex: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  border-radius: 2px;
  padding: 0;
  white-space: nowrap;
  font-size: 15px;
  line-height: 19px;
  font-weight: 500;
  border: none;
  cursor: pointer;
  color: #ffffff;
  background-color: transparent;
  width: 200px;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 16px;
  padding-right: 12px;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-bottom: 4px;
  color: #ffffff;
}

.emotion-4,
.emotion-4:hover,
.emotion-4:visited {
  background-color: #808080;
}

.emotion-4:hover {
  background-color: #b3b3b3;
}

.emotion-4[aria-disabled=true] {
  background-color: #808080;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-right: 48px;
  padding-top: 12px;
  padding-bottom: 12px;
}

.emotion-6 {
  font-weight: 700;
  color: #ffffff;
  padding-bottom: 4px;
}

.emotion-7 {
  color: #b3b3b3;
}

.emotion-20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 16px;
  width: 212px;
  position: relative;
}

.emotion-21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

.emotion-22 {
  height: 100%;
  -webkit-flex: 3 0 auto;
  -ms-flex: 3 0 auto;
  flex: 3 0 auto;
  background-color: #009f22;
}

.emotion-22:first-of-type {
  border-top-left-radius: 4px;
  border-bottom-left-radius: 4px;
}

.emotion-22:last-of-type {
  border-top-right-radius: 4px;
  border-bottom-right-radius: 4px;
}

<div>
  <h3
    className="emotion-0"
  >
    Job: 
    image_similarity.json
  </h3>
  <div
    className="emotion-1"
  >
    <div
      className="emotion-2"
    >
      <div
        className="emotion-3"
      >
        <button
          aria-label="Toggle Dropdown Menu"
          className="emotion-4"
          onBlur={[Function]}
          onClick={[Function]}
          type="button"
        >
          Modify Job
          <svg
            data-file-name="chevron.svg"
            height={20}
          />
        </button>
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        ID
        :
      </div>
      <div
        className="emotion-7"
      >
        82d5308b-67c2-1433-8fef-0a580a000956
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Status
        :
      </div>
      <div
        className="emotion-7"
      >
        InProgress
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Priority
        :
      </div>
      <div
        className="emotion-7"
      >
        100
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        # Assets
        :
      </div>
      <div
        className="emotion-7"
      >
        4
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Progress
        :
      </div>
      <div
        className="emotion-7"
      >
        <div
          aria-label="Progress Bar"
          className="emotion-20"
          onKeyPress={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="button"
          tabIndex="0"
        >
          <div
            className="emotion-21"
          >
            <div
              className="emotion-22"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<JobDetails /> should render properly with an action confirmation 1`] = `
.emotion-0 {
  font-size: 17px;
  line-height: 20px;
  font-weight: 500;
  padding-top: 16px;
  padding-bottom: 16px;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  background-color: #dbecff;
  border-radius: 2px;
  padding: 12px;
  padding-left: 16px;
  padding-right: 16px;
}

.emotion-3 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding-left: 12px;
  color: #191919;
  font-weight: 500;
  white-space: pre-line;
}

.emotion-3 a {
  color: #191919;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  margin-left: 4px;
}

.emotion-3 a:hover {
  color: #3A5DFF;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-right: 48px;
  padding-top: 12px;
  padding-bottom: 12px;
}

.emotion-6 {
  font-weight: 700;
  color: #ffffff;
  padding-bottom: 4px;
}

.emotion-7 {
  color: #b3b3b3;
}

.emotion-20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 16px;
  width: 212px;
  position: relative;
}

.emotion-21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

.emotion-22 {
  height: 100%;
  -webkit-flex: 3 0 auto;
  -ms-flex: 3 0 auto;
  flex: 3 0 auto;
  background-color: #009f22;
}

.emotion-22:first-of-type {
  border-top-left-radius: 4px;
  border-bottom-left-radius: 4px;
}

.emotion-22:last-of-type {
  border-top-right-radius: 4px;
  border-bottom-right-radius: 4px;
}

<div>
  <h3
    className="emotion-0"
  >
    Job: 
    image_similarity.json
  </h3>
  <div
    className="emotion-1"
  >
    <div
      className="emotion-2"
    >
      <svg
        color="#0074f5"
        data-file-name="information.svg"
        height={20}
      />
      <div
        className="emotion-3"
        role="alert"
      >
        Retry All Failures
      </div>
    </div>
  </div>
  <div
    className="emotion-4"
  >
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        ID
        :
      </div>
      <div
        className="emotion-7"
      >
        82d5308b-67c2-1433-8fef-0a580a000956
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Status
        :
      </div>
      <div
        className="emotion-7"
      >
        Success
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Priority
        :
      </div>
      <div
        className="emotion-7"
      >
        100
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        # Assets
        :
      </div>
      <div
        className="emotion-7"
      >
        4
      </div>
    </div>
    <div
      className="emotion-5"
    >
      <div
        className="emotion-6"
      >
        Progress
        :
      </div>
      <div
        className="emotion-7"
      >
        <div
          aria-label="Progress Bar"
          className="emotion-20"
          onKeyPress={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="button"
          tabIndex="0"
        >
          <div
            className="emotion-21"
          >
            <div
              className="emotion-22"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
